src/main/resources/referenceImageIDTemplateWithManagedDisk.json

```diff
@@ -74,7 +74,7 @@
           "vmSize": "[variables('vmSize')]"
         },
         "osProfile": {
-          "computername": "[concat(variables('vmName'), copyIndex())]",
+          "computerName": "[concat(variables('vmName'), copyIndex())]",
```

Comment:
api docs have it camel cased and IDE was complaining about it, no harm in fixing even though it works either way